waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -97,6 +98,17 @@ alexGetByte (_, [], (currChar:remainingSource)) = case encodeChar currChar of
 alexInputPrevChar :: AlexInput -> Char
 alexInputPrevChar (prevChar, _, _) = prevChar
 
+-- | Tokenizes the given string. 
+alexScanTokens :: String -> Maybe [Parser Token]
+alexScanTokens str = go ('\n', [], str)
+  where
+    go input@(_, _, lexeme) = case alexScan input 0 of
+      AlexEOF -> Just []
+      AlexError _ -> Nothing
+      AlexSkip rest _ -> go rest
+      AlexToken rest len act -> (currToken:) <$> go rest
+        where currToken = act (take len lexeme)
+
```

Comment:
@Martinsos I decided to make it a bit more general/reusable than the version we had earlier today.

This function tokenizes any string according to Wasp's lexical grammar and returns a list of `Parser Token`. I then later ensure that this result corresponds to a single identifier.

Reply:
Ok, makes sense, I think that is a good way to do it!